apps/frontend/src/shared-research/__tests__/api.test.ts

```diff
@@ -84,12 +84,14 @@ describe('getResearchOutputs', () => {
   it('builds a multiple filter query', async () => {
     await getResearchOutputs(mockIndex, {
       ...options,
-      filters: new Set<ResearchOutputType>(['Article', 'Proposal']),
+      filters: new Set<ResearchOutputType>(['Article', 'Grant Document']),
```

Comment:
so if the Proposal is still being left should we not keep this test and add a new one to show how grant documents are searched?

Reply:
You cannot search by the Proposal. It's removed on the FE
![image](https://user-images.githubusercontent.com/9073485/143023337-b58c3b7b-83f5-4d8b-89a3-13dbd244ac7f.png)